examples/src/main/scala/TracingExample.scala

```diff
@@ -50,17 +64,32 @@ object Work {
 }
 
 object TracingExample extends IOApp.Simple {
-  def tracerResource: Resource[IO, Tracer[IO]] =
+  def globalOtel4s: Resource[IO, Otel4s[IO]] =
     Resource
       .eval(IO(GlobalOpenTelemetry.get))
       .evalMap(OtelJava.forSync[IO])
-      .evalMap(_.tracerProvider.get("Example"))
 
   def run: IO[Unit] = {
-    tracerResource.use { implicit tracer: Tracer[IO] =>
-      val resource: Resource[IO, Unit] =
-        Resource.make(IO.sleep(50.millis))(_ => IO.sleep(100.millis))
-      tracer.resourceSpan("resource")(resource).surround(Work[IO].doWork)
+    globalOtel4s.use { otel4s: Otel4s[IO] =>
+      implicit val textMapProp: TextMapPropagator[IO] =
+        otel4s.propagators.textMapPropagator
```

Comment:
Get implicitly from an implicit otel4s?  Too implicit?

Reply:
Implicit conversion is fine by me. It would also be nice to provide a meaningful `@implicitNotFound` message.